From James.Bottomley@HansenPartnership.com  Wed Feb 20 14:22:51 2008
From: James Bottomley <James.Bottomley@HansenPartnership.com>
Date: Wed, 20 Feb 2008 15:14:24 -0600
Subject: PCI: remove parisc consumer of the pci global_list
To: Greg KH <greg@kroah.com>
Cc: Parisc List <linux-parisc@vger.kernel.org>, linux-kernel <linux-kernel@vger.kernel.org>
Message-ID: <1203542065.3109.42.camel@localhost.localdomain>

Remove the parisc usage of the global_list, as it's not needed anymore.

Signed-off-by: James Bottomley <James.Bottomley@HansenPartnership.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/parisc/dino.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/parisc/dino.c
+++ b/drivers/parisc/dino.c
@@ -496,7 +496,6 @@ dino_card_setup(struct pci_bus *bus, voi
 		list_for_each_safe(ln, tmp_ln, &bus->devices) {
 			struct pci_dev *dev = pci_dev_b(ln);
 
-			list_del(&dev->global_list);
 			list_del(&dev->bus_list);
 		}
 			
